doc-content/jbpm-docs/src/main/asciidoc/BPMN2/con_custom-tasks-overview.adoc

```diff
@@ -0,0 +1,216 @@
+
+* *Endpoint*: Endpoint location of the web service to invoke.
+* *Interface*: Name of a service, such as `Weather`.
+* *Mode*: Mode of a service, such as `SYNC`, `ASYNC`, or `ONEWAY`.
+* *Namespace*: Namespace of the web service, such as `http://ws.cdyne.com/WeatherWS/`.
+* *Operation*: Method name to call.
+* *Parameter*: Object or array to be sent for the operation.
+* *Url*: URL of the web service, such as `http://wsf.cdyne.com/WeatherWS/Weather.asmx?WSDL`.
+
+You can add the following data output in *Data Outputs and Assignments* to store the output of the task execution:
+
+* *Result*: Output variable (object type) of the web service task.
+
+.Milestone
+A milestone represents a single point of achievement within a process instance. You can use milestones to flag certain events to trigger other tasks or track the progress of the process.
+
+image::BPMN2/bpmn-milestone.png[]
+
+Milestones are useful for Key Performance Indicator (KPI) tracking or for identifying the tasks that are still to be completed. Milestones can occur at the end of a stage in a process or they can be the result of achieving other milestones.
+
+Milestones can reach the following states during process execution:
+
+* `Active`: A milestone condition has been defined for the milestone node but it has not been met.
+* `Completed`: A milestone condition has been met (if applicable), the milestone has been achieved, and the process can proceed to the next task or can end.
+
+You can click *Assignments* in the *Properties* panel to open the *Milestone Data I/O* window. In the *Milestone Data I/O* window, you can configure the data input and output as required. For example, to execute a milestone, enter the following data inputs in *Data Inputs and Assignments* fields:
+
+* *Condition*: Condition for the milestone to meet. For example, you can enter a Java expression (string data type) that uses a process variable.
+
+.DecisionTask
+A decision task is used to execute a DMN diagram and invoke a decision engine service from a process. By default, a decision task maps to the DMN decision.
+
+image::BPMN2/bpmn-decision-task-custom.png[]
+
+You can use decision tasks to make an operational decision in a process. Decision tasks are useful for identifying key decisions in a process that need to be made.
+
+You can click *Assignments* in the *Properties* panel to open the *Decision Task Data I/O* window. In the *Decision Task Data I/O* window, you can configure the data input and output as required. For example, to execute a decision task, enter the following data inputs in *Data Inputs and Assignments* fields:
+
+* *Decision*: Decision for a process to make.
+* *Language*: Language of the DMN model, such as `FEEL` or `S-FEEL`.
```

Comment:
Actually based on the code here:
https://github.com/kiegroup/jbpm/blob/402894e7b9e5cd1ba2efcf31ac62224ec1524472/jbpm-workitems/jbpm-workitems-bpmn2/src/main/java/org/jbpm/process/workitem/bpmn2/AbstractRuleTaskHandler.java#L130-L161
For decision, there is currently only one language possible and that is DMN.
Also, this value is a default one for the Decision Task, so `Language` parameter doesn't have to be specified.

Reply:
Done, description changed to _Language of the DMN model, defaults to DMN._

Reply:
It is not the language of the DMN model, it is a language of the decision task.

Reply:
Done, changed to Language of the decision task, defaults to DMN.